src/main/java/com/gw2auth/oauth2/server/service/OAuth2ClientApiVersion.java

```diff
@@ -0,0 +1,32 @@
+package com.gw2auth.oauth2.server.service;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum OAuth2ClientApiVersion {
+
+    V0(0),
+    V1(1);
+
+    public static final OAuth2ClientApiVersion CURRENT = V0;
```

Comment:
It will remain at V0 until
- this version has been tested intensively in the field, in case I need to rollback
- the developer wiki has been updated for V1